odoo90/scripts/build-image.sh

```diff
@@ -32,26 +32,7 @@ NPM_DEPENDS="less \
              jshint"
 PIP_OPTS="--upgrade \
           --no-cache-dir"
-PIP_DEPENDS_EXTRA="pyyaml \
-                   pillow \
-                   pillow-pil \
-                   M2Crypto \
-                   GeoIP \
-                   SOAPpy \
-                   suds \
-                   lxml \
-                   pandas \
-                   qrcode \
-                   xmltodict \
-                   flake8 \
-                   pylint-mccabe \
-                   PyWebDAV \
-                   mygengo \
-                   recaptcha-client \
-                   egenix-mx-base \
-                   hg+https://bitbucket.org/birkenfeld/sphinx-contrib@default#egg=sphinxcontrib-youtube&subdirectory=youtube \
-                   git+https://github.com/vauxoo/pylint-odoo@master#egg=pylint-odoo \
-                   git+https://github.com/vauxoo/panama-dv@master#egg=ruc"
+PIP_DEPENDS_EXTRA=$(cat ${DEPENDENCIES_FILE})
```

Comment:
Be careful with this.

If you have a `requirements.txt` with comments like these (which are normal):

```
req1
# this req is for Y
req2
```

Then the result of cat'ing that file in a subshell will be:

```
req1 # this req is for Y req2
```

Which is not what you wanted.